src/kernel/src/arch/aarch64/address.rs

```diff
@@ -249,7 +249,7 @@ impl PhysAddr {
     }
 
     pub fn kernel_vaddr(&self) -> VirtAddr {
-        phys_to_virt(*self) // maybe???
```

Comment:
Either be more specific or just make the change.  Or maybe  turn that into XXX so you can find it later?

Reply:
I actually meant to remove this comment. That functionality has already been implemented.